source/appModules/soffice.py

```diff
@@ -449,13 +457,36 @@ class SymphonyDocument(CompoundDocument):
 	# variables used for handling announcements resulting from gestures
 	GESTURE_ANNOUNCEMENT_TIMEOUT = 2.0
```

Comment:
Could you please add the unit here?
```suggestion
	GESTURE_ANNOUNCEMENT_TIMEOUT = 2.0  # Seconds
```